cmd/admin-handlers.go

```diff
@@ -501,7 +502,7 @@ func (adminAPI adminAPIHandlers) HealObjectHandler(w http.ResponseWriter, r *htt
 	writeSuccessResponseHeadersOnly(w)
 }
 
-// HealFormatHandler - POST /?heal
+// HealFormatHandler - POST /?heal&dry-run=[yes|no]
```

Comment:
Ok, I shall make the necessary changes to the other heal APIs that support this too.